pages/moneyspending.js

```diff
@@ -0,0 +1,9 @@
+import TransactionForm from "../components/TransactionForm";
+
+export default function Moneyspending({ onAddSpendingValues }) {
+  return (
+    <>
+      <TransactionForm title="Ausgaben" onAddEntry={onAddSpendingValues} />
+    </>
+  );
+}
```

Comment:
Do you use the `<TransactionForm>` component more than once? 
Otherwise I would not outsource it but render the form directly. This way you don't have to pass on the properties unnecessarily - but this s just a tiny suggestion. Furthermore the `<></>` are not necessary 😉 


```suggestion
import TransactionForm from "../components/TransactionForm";

export default function Moneyspending({ onAddSpendingValues }) {
  return (
      <TransactionForm title="Ausgaben" onAddEntry={onAddSpendingValues} />
  );
}
```